### PR TITLE
WIP - Update anti-spam-message-headers.md

### PR DESCRIPTION
**Pull request by Sherd21**
adding definition of CAT flag in "X-Forefront-Antispam-Report message header fields" section and a link to a page to explain order of policies to be applied.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SecurityCompliance/anti-spam-message-headers.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ After accessing the message header information, search for **X-Forefront-Antispa
 |SFV:NSPM|The message was marked as non-spam and was sent to the intended recipients.|
 |H: [helostring]|The HELO or EHLO string of the connecting mail server.|
 |PTR: [ReverseDNS]|The PTR record, or pointer record, of the sending IP address, also known as the reverse DNS address.|
+|CAT:|The category of protection policy, applied to the message: <br/>• MALW: Malware <br/>• PHSH: Phishing <br/>• HSPM: High confidence spam <br/>• SPOOF: Spoofing <br/>• SPM: Spam <br/>• BULK: Bulk <br/>• DIMP: Domain Impersonation <br/>• UIMP: User Impersonation <br/>Potentially, an incoming message may be flagged by multiple forms of protection, and multiple detection scans. Policies have different priority and the one at the highest priority will apply. [What policy applies when multiple protection methods and detection scans run on your email](https://docs.microsoft.com/en-us/office365/securitycompliance/how-policies-and-protections-are-combined)|
```
Comment (nenonix) [inline SecurityCompliance/anti-spam-message-headers.md]:
```suggestion
|CAT:|The category of protection policy, applied to the message: <br/>• MALW: Malware <br/>• PHSH: Phishing <br/>• HSPM: High confidence spam <br/>• SPOOF: Spoofing <br/>• SPM: Spam <br/>• BULK: Bulk <br/>• DIMP: Domain Impersonation <br/>• UIMP: User Impersonation <br/>Potentially, an incoming message may be flagged by multiple forms of protection, and multiple detection scans. Policies have different priorities, and the one at the highest priority will apply. See [What policy applies when multiple protection methods and detection scans run on your email](https://docs.microsoft.com/office365/securitycompliance/how-policies-and-protections-are-combined).|
```

**Comment by scanum** [comment]
@KCCross Hi KC, changes look correct. Could you please confirm and merge? Thanks!

**Comment by KCCross** [comment]
Also needs review by David Fantham.

**Comment by KCCross** [comment]
Tracy, there's still a merge conflict in here and it looks like the bullets are part of the conflict. Please resolve that conflict and merge the content. Contact me if it gives you any grief.

File: SecurityCompliance/anti-spam-message-headers.md (hunk 1 above)
Comment (KCCross) [reply SecurityCompliance/anti-spam-message-headers.md]: You must remove HTML style bullets from the table. If you're having trouble rethinking how to format this, I've requested help internally.

**Comment by KCCross** [comment]
Closing this PR. Changes incorporated and published through https://github.com/MicrosoftDocs/OfficeDocs-O365SecComp-pr/pull/2026

**Pull request closed without merging**